codec/src/main/java/io/netty/handler/codec/DefaultHeaders.java

```diff
@@ -67,6 +67,11 @@ private static int index(int hash) {
     int size;
 
     public interface NameValidator<T> {
+        /**
+         * Throws a {@link RuntimeException} if the name is invalid.
```

Comment:
nit: probably don't need the description since it's redundant with the `@throws` tag.  Also, put `@param` before the `@throws`?